back/app/controllers/web_api/v1/projects_allowed_input_topics_controller.rb

```diff
@@ -0,0 +1,59 @@
+class WebApi::V1::ProjectsAllowedInputTopicsController < ApplicationController
+  before_action :set_projects_allowed_input_topic, only: %i[show reorder destroy]
+  #skip_before_action :authenticate_user, only: %i[index, show]
```

Comment:
It seems indeed this controller is not used by citizens, but still please try to test (first locally, then on staging) all projects/topics-related functionality after signing out.